### ISSUE TITLE
Is this still being maintained?

### ISSUE THREAD

**Issue by amimas** [lukeapage/node-markdown-spellcheck#122]
Just want to know if this is a stale project.

**Comment by amimas** [comment]
@lukeapage - would you be able to provide a quick update? It's understandable if you don't have the bandwidth to maintain this project. Maybe others can help with the maintenance.

**Comment by lukeapage** [comment]
Hi, not really.. the problem is I don’t use it. @robatwilliams also has permission to merge as I believe they use it

**Comment by robatwilliams** [comment]
It is in use by the company I work for, as part of our website's publishing toolchain. While that toolchain is in regular use, it's "done" and working - not undergoing development at the moment. If an issue arises in this project that affects that toolchain, it'd probably be seen to fairly quickly.

If there's anything in particular that needs fixing, I'd be happy to try and find if we have someone who's not busy and who would be able to merge any PRs and do a release.

**Comment by dialex** [comment]
@robatwilliams Hi there, there's something that needs fixing, yes, a moderate security flaw was discovered on `marked`, a dependency of this project. [Source](https://www.npmjs.com/advisories/812)

Please update it to [v0.7.0](https://www.npmjs.com/package/marked). Would it help if I made a PR?

**Comment by robatwilliams** [comment]
Hi, thanks for bringing this up. I'm not the owner of the repo, so didn't receive a security notification from GitHub.

We specify `^0.3.5`, anyone installing it will get `0.7.0` so I'm not sure this is much of a concern for this project. Correct me if I'm wrong. So I don't see reason to do a release right now. Thanks for the PR.

I see there have been many [breaking changes](https://github.com/markedjs/marked/releases) to `marked` since the 0.3.5 version that was originally installed in this project. I'm not sure how significant they are, given that the project isn't using semantic versioning.

**Comment by dialex** [comment]
> We specify ^0.3.5, anyone installing it will get 0.7.0 so I'm not sure this is much of a concern for this project. Correct me if I'm wrong. So I don't see reason to do a release right now. Thanks for the PR.

Not really, with `^0.3.5` you will get all patch updates -- in this specific case you will get up to `0.3.19`. The security flaw was only patched at `0.6.3`. In the meantime, we are at risk.

<img width="611" alt="Screenshot 2019-09-19 at 19 11 50" src="https://user-images.githubusercontent.com/2140450/65269581-606a0a80-db11-11e9-9186-21f89cb9be23.png">

**Comment by robatwilliams** [comment]
Let's continue this discussion on the PR, as it's not the subject of this issue.

**Comment by armano2** [comment]
hi, i did some work on updating all dependencies, and migrating some code, and overall cleanup

https://github.com/lukeapage/node-markdown-spellcheck/pull/135

some of tests on master do not work (due to weird usage of promises)

**Comment by jeffgore00** [comment]
Just FYI, I found what appears to be a maintained library that matches the functionality of this one:
https://www.npmjs.com/package/spellchecker-cli
